pkg/scrape/vrbangers.go

```diff
@@ -111,36 +106,37 @@ func VRBangersSite(wg *sync.WaitGroup, updateSite bool, knownScenes []string, ou
 		}
 
 		// Cast
-		e.ForEach(`div.video-item-info--starring-download a`, func(id int, e *colly.HTMLElement) {
+		e.ForEach(`div.video-item__info-starring div.ellipsis a`, func(id int, e *colly.HTMLElement) {
 			sc.Cast = append(sc.Cast, strings.TrimSpace(strings.Replace(e.Text, ",", "", -1)))
 		})
 
 		out <- sc
 	})
 
-	siteCollector.OnHTML(`div.video-item-info--title a`, func(e *colly.HTMLElement) {
+	siteCollector.OnHTML(`div.video-item-info-title a`, func(e *colly.HTMLElement) {
 		// Some index pages have links to german language scene pages:
 		// https://vrbangers.com/video/sensual-seduction/?lang=de
 		// This will strip out the query params... gross
 		url := strings.Split(e.Attr("href"), "?")[0]
 		sceneURL := e.Request.AbsoluteURL(url)
 
+		// Sneak Peek 2020 URL will crash the scraper
 		if !funk.ContainsString(knownScenes, sceneURL) {
 			sceneCollector.Visit(sceneURL)
 		}
 	})
 
-	siteCollector.OnHTML(`.pagination a`, func(e *colly.HTMLElement) {
+	siteCollector.OnHTML(`.pagination-next a`, func(e *colly.HTMLElement) {
 		// Some index pages have links to german language scene pages:
 		// https://vrbangers.com/video/sensual-seduction/?lang=de
 		// This will strip out the query params... gross
-		url := strings.Split(e.Attr("href"), "?")[0]
-		pageURL := e.Request.AbsoluteURL(url)
+		//url := strings.Split(e.Attr("href"), "?")[0]
+		pageURL := e.Request.AbsoluteURL(e.Attr("href"))
 
 		siteCollector.Visit(pageURL)
 	})
 
-	siteCollector.Visit(URL)
+	siteCollector.Visit(URL + "?sort=latest")
```

Comment:
A few scenes (_As Thick As Thieves_, _Bachelor No More_ & _Sorority Hookup 5_) are apparently considered bonus videos and show up in the list without an extra parameter:
```suggestion
	siteCollector.Visit(URL + "?sort=latest&bonus-video=1")
```